src/pages/Confirmation.jsx

```diff
@@ -0,0 +1,149 @@
+import React, {useEffect, useState} from 'react';
+import Rating from 'react-rating';
+import styled from "styled-components";
+import Button from "../components/Button";
+import Card from "../components/QuestionCard";
+import axios from "../axios-instance";
+import StarOn from "assets/StarOnIcon.svg"
+import StarOff from "assets/StarOffIcon.svg"
+import {v4 as uuidv4} from "uuid";
+import { motion } from "framer-motion"
```

Comment:
```suggestion
import React, {useEffect, useState} from 'react';
import styled from 'styled-components';
import Rating from 'react-rating';
import {v4 as uuidv4} from 'uuid';
import { motion } from 'framer-motion';

import axios from 'axios-instance';

import Button from 'components/Button';
import Card from 'components/QuestionCard';

import StarOn from 'assets/StarOnIcon.svg';
import StarOff from 'assets/StarOffIcon.svg';
```
Dobry zwyczaj oddzielać zewnętrzne paczki od komponentów i od assetów